### PR TITLE
Implemented cummax on torch frontend #13037

### PR DESCRIPTION
**Pull request by tvsharish**
I have implemented cummax on torch frontend . I have also created corresponding testing function. Presently, torch backend of cummax is not implemented. Can I create a new issue on its implementation?

### CODE REVIEW

File: ivy/functional/frontends/torch/miscellaneous_ops.py [hunk 1]
```diff
@@ -75,6 +75,12 @@ def tril_indices(row, col, offset=0, *, dtype=ivy.int64, device="cpu", layout=No
     return ivy.stack(ivy.nonzero(sample_matrix)).astype(dtype)
 
 
+@to_ivy_arrays_and_back
+def cummax(input, dim, *, out=None):
+
+    return ivy.cummax(input, axis=dim, out=out)
```
Comment (zaeemansari70) [inline ivy/functional/frontends/torch/miscellaneous_ops.py:81]: This is a dummy call as there is no backend function

**Review by zaeemansari70 — COMMENTED** [review]
Hi @tvsharish ,! 
Thank you so much for contributing to ivy! 🙂

This PR of yours is not functional as the frontend does not have a backend which it can call. If the issue is open you can assign the issue to yourself and start working on the backend, if not you have to create an issue with a description explaining that why should we add this function in Ivy :slightly_smiling_face: 

This can only be reviewed once the backend has been added, 
Thanks! :slightly_smiling_face: 

Do let me know if you have any other questions!
Thanks!

**Comment by tvsharish** [comment]
Hi, Since all the backends of cummax is implemented. I have written ```test_torch_cummax``` function on torch frontend.  This will close issue #12998 which is subtask of #3612 .

**Comment by MahmoudAshraf97** [comment]
Hi @tvsharish , please open a new PR containing only the relevant frontend changes

**Pull request closed without merging**